docs/reactivity/key-concepts.md

```diff
@@ -0,0 +1,71 @@
+# Key Concepts of Reactivity
+
+## Working with reactive data streams
+
+Reactive data streams are a fundamental concept in Oryx, and play a crucial role in managing and manipulating data in real-time. Oryx prefers Observables over Promises as they allow to keep the application in sync during the full application lifetime. This is particular helpful in experiences that remain active during some time, for example in a Single Page Application (SPA).
```

Comment:
> Oryx prefers Observables over Promises as they allow to keep the application in sync during the full application lifetime.

So without Observables there is no way to keep the application in sync? Not sure I can fully agree with this statement. Maybe we can rephrase and say it's much easier with Observables and even provide some comparison.

Reply:
I'll rephrase